### PR TITLE
Result page

### LINKED ISSUE #11: [Feature 10] Result Page
Create a result page that works with switch pages and interactor and presents results to the user

### PR DESCRIPTION
**Pull request by takeachangs**
Results page with examples of recipes. When clicked it will open the full recipes on the browser. Will make it into a loop with inputs from recipe builder presenter once it is done.

### CODE REVIEW

File: src/main/java/presenters/ButtonCreator.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class ButtonCreator {
```
Comment (EricKarpovits) [inline src/main/java/presenters/ButtonCreator.java]: Add docstrings for the class and methods please

File: src/main/java/presenters/ResultsPageFrame.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import javax.swing.border.Border;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+
+public class ResultsPageFrame extends JFrame implements RecipeBuilderPresenter {
```
Comment (EricKarpovits) [inline src/main/java/presenters/ResultsPageFrame.java]: Add docstrings for the class and methods please

File: src/main/java/presenters/ResultsPageLauncher.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package presenters;
+
+
+import javax.swing.*;
+import java.net.MalformedURLException;
+
+public class ResultsPageLauncher {
```
Comment (EricKarpovits) [inline src/main/java/presenters/ResultsPageLauncher.java]: Add docstrings for the class and methods please

**Review by EricKarpovits — CHANGES_REQUESTED** [review]

**Review by takeachangs — COMMENTED** [review]
Added docstrings

**Review by EricKarpovits — APPROVED** [review]
Great work @takeachangs!

**Review by fonuxxine — APPROVED** [review]
Your docstrings explain nicely different parts of your code.

File: src/main/java/UI/resultPage/ResultsPageMain.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
+/*package presenters;
```
Comment (fishyvishy) [inline src/main/java/UI/resultPage/ResultsPageMain.java]: should probably delete this class altogether.

**Review by fishyvishy — APPROVED** [review]
Well done. Good implementation using MVVM.

**Pull request merged**